vcpkg/consume/manifest-mode.md

```diff
@@ -0,0 +1,328 @@
+
+In a new folder, create a source file named `main.cxx` with these contents:
+
+:::code language="cxx" source="../examples/snippets/manifest-mode-cmake/main.cxx":::
+
+The code references the open-source libraries: `cxxopts`, `fmt`, and `range-v3`; which are all
+available in the vcpkg public registry at <https://github.com/Microsoft/vcpkg>.
+
+To declare these dependencies, create a file named `vcpkg.json` in the same directory as your project:
+
+`vcpkg.json`:
+
+:::code language="json" source="../examples/snippets/manifest-mode-cmake/vcpkg.json":::
+
+You only need to specify your direct dependencies in the `"dependencies"` list. When it runs, vcpkg
+resolves and installs any required transitive dependencies.
+
+## 2 - Integrate vcpkg with your build system
+
+In this step we show you how to integrate vcpkg with CMake or MSBuild, so that your project dependencies get automatically installed or restored whenever you build the project. 
+
+If you're using a different build system, skip to the next step: [3 - Install the manifest dependencies].
+
+### [MSBuild](#tab/msbuild)
+
+> [!div class="nextstepaction"]
+> [Learn more about using vcpkg from MSBuild](../users/buildsystems/msbuild-integration.md)
+
+To use [vcpkg in your MSBuild projects](../users/buildsystems/msbuild-integration.md), run the
+following command:
+
+```Console
+vcpkg integrate install
+```
+
+You only need to run the [`vcpkg integrate install`](../commands/integrate.md#vcpkg-integrate-install)
+command the first time you want to enable MSBuild integration. This enables MSBuild integration for all
+your existing and future projects. Use [`vcpkg integrate remove`](../commands/integrate.md#vcpkg-integrate-remove)
+to remove MSBuild system-wide integration.
+
+This integration method automatically adds vcpkg-installed packages to the following project properties:
+`Include Directories`, `Link Directories`, and `Link Libraries`. Additionally, this creates a post-build action
+that ensures that any required DLLs are copied into the build output folder. This works for all solutions and
+projects using Visual Studio 2015 or newer.
+
+### [CMake](#tab/cmake)
+
+> [!div class="nextstepaction"]
+> [Learn more about using vcpkg from CMake](../users/buildsystems/cmake-integration.md)
+
+To use [vcpkg in your CMake projects](../users/buildsystems/cmake-integration.md), you need to set the
+`CMAKE_TOOLCHAIN_FILE` variable to use vcpkg's CMake toolchain file. The vcpkg toolchain is in
+`%VCPKG_ROOT%/scripts/buildsystems/vcpkg.cmake`, where `%VCPKG_ROOT%` is your vcpkg installation path.
+
+To set the toolchain file use any of these methods:
+
+* Set the `CMAKE_TOOLCHAIN_FILE` in your CMakePresets.json file.
+* Pass `-DCMAKE_TOOLCHAIN_FILE=<path/to/vcpkg>/scripts/buildsystems/vcpkg.cmake` as a parameter in your
+CMake configure call.
+* Set the `CMAKE_TOOLCHAIN_FILE` CMake variable before the first call to `project()` in your
+`CMakeLists.txt` file.
+
+---
+
+## 3 - Install dependencies
```

Comment:
```suggestion
## 3 - Install dependencies (non CMake or MSBuild)
```

I know you said this in the first sentence but I think it'll help a user 'scanning' the document to see they can probably skip it in the doc.

It might also be good to find a way to reorder this such that people have to skip over msbuild or cmake to get here since we expect those two to be more common (by effectively combining step 3 and 4). But that's extreme nitpick territory.

Reply:
I'll split this document into separate workflows (projects with integration vs projects without it) in a future PR